tick/base/__init__.py

```diff
@@ -18,7 +18,7 @@ def _set_mpl_backend():
 
 _set_mpl_backend()
 
-from .array import *
+from tick.array import *
```

Comment:
Why not but was this necessary?

Reply:
array is no longer a subdirectory of base so I believe so